src/__tests__/__snapshots__/role-helpers.js.snap

```diff
@@ -52,32 +52,6 @@ Name "":
   data-testid="a-article"
 />
 
---------------------------------------------------
-command:
-
-Name "":
```

Comment:
I remember that `menuitem` has been deprecated, I'm hesitating whether this is a breaking change on our end or not. People who used `menuitem` before will now not get `menuitem` role or `command` role.
Any thoughts about this? If we decide to not push it as a major (and I don't think we should), we should clearly state that this has been deprecated due to the ARIA spec.

Reply:
Yeah I'm not sure how best to proceed. The situation as I understand it:

- The `menuitem` element is deprecated. REF: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menuitem and https://caniuse.com/?search=menuitem
- No browsers other than Firefox ever supported the element officially.
- Firefox stopped supporting the element in Firefox 85. For perspective, the latest current version of Firefox is 114 and the minimum supported version by `dom-testing-library` [is 91 according to the browserslist config](https://github.com/testing-library/dom-testing-library/blob/main/package.json#L34).
- In WAI ARIA 1.1, pre deprecation of the element, the role `menuitem` and the abstract role `command` were associated with the element. REFs: https://www.w3.org/TR/wai-aria-1.1/#menuitem and https://www.w3.org/TR/wai-aria-1.1/#command
- In WAI ARIA 1.2 the `menuitem` element and it's prior role associations were dropped prior to 2 March 2021 Candidate Snapshot (REF: https://www.w3.org/TR/2021/CR-wai-aria-1.2-20210302/)
- In HTML-ARIA the `menuitem` and `command` roles for the `menuitem` element were dropped from the element in 30 May 2019 working draft (https://www.w3.org/TR/2019/WD-html-aria-20190530/) and the element was later removed altogether before reaching the candidate stage of the spec proposal.

Ideally no-one should ever really used this element, and they certainly shouldn't be now. For most standards this element has either been dropped years ago, or never officially recognised.

I suspect it would be reasonably safe to go for a minor change, with a small risk of there being complaint. A major would be a certain measure, but feels potentially overkill for something that has never been supported by the standard suite of major browsers.

Reply:
This said, given the number of other changes this will bring to testing-library, the PR as a whole is hard to not consider as a breaking change - WAI ARIA 1.1 to 1.2 is not backward compatible IMO.